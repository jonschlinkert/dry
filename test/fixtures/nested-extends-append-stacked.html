{% extends "nested-extends-append-stacked" %}
{% block append="one" %}
one: "nested-extends-append-stacked.html"
{% endblock %}
