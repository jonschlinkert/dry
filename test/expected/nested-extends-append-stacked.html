
one: "blocks/parent-append-stacked.html"
one: "blocks/nested-extends-append-stacked.html"
one: "nested-extends-append-stacked.html"